packages/web/src/components/search/CategorySearch.js

```diff
@@ -410,11 +410,15 @@ class CategorySearch extends Component {
 			} // prettier-ignore
 			: query;
 
-		props.setQueryOptions(this.props.componentId, {
+		const finalQueryOptions = componentId === this.internalComponent ? {
```

Comment:
It's best if we undo this - because we won't be supporting `defaultQuery` in search components.

Reply:
Updated